noCircularImportsRule.ts

```diff
@@ -1,8 +1,15 @@
 import { relative, sep } from 'path';
 import * as Lint from 'tslint';
-import { IOptions } from 'tslint/lib/language/rule/rule';
 import * as ts from 'typescript';
 
+interface Options {
+  /** @internal */
```

Comment:
What's this for?

Reply:
Yeah I was a little confused how to represent this. It's not common for the `interface Options` on a rule to be different from the config the rule is allowed to take in.

`@internal` is recognized by TS and seems to be coming up as part of [TSDoc](https://github.com/Microsoft/tsdoc/issues/8), so I figured it was appropriate here. If you change or remove this I won't be offended at all 😉